.github/workflows/pr-vrt-poc-comment.yml

```
@@ -56,4 +64,4 @@ jobs:
           message: |
             VR Regression Report:
             - status: ${{ github.event.workflow_run.conclusion }}
-            - check whole [report](${{github.event.workflow_run.workflow_url}}) to review visual differences
+            - check whole [report](${{github.event.workflow_run.html_url}}) to review visual differences
```

I guess this should lead to correct page 🤔 

```suggestion
            - check whole [report](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.event.workflow_run.id }}) to review visual differences
```

---

yeah, the docs are not helping in this regard that's why I added to debug step, after we get that we can adjust for good. my hopes are that the html_url would do the trick, if not, we will try your proposal.